src/manifolds/PowerManifold.jl

```diff
@@ -64,22 +64,40 @@ By default, the [`ArrayPowerRepresentation`](@ref) of points
 and tangent vectors is used, although a different one, for example
 [`NestedPowerRepresentation`](@ref), can be given as the second argument to the
 constructor.
+When `M` is a `PowerManifold` (not any [`AbstractPowerManifold`](@ref)) itself, given
+dimensions will be appended to the dimensions already present, for example
+`PowerManifold(PowerManifold(Sphere(2), 2), 3)` is equivalent to
+`PowerManifold(Sphere(2), 2, 3)`. This feature preserves the representation of the inner
```

Comment:
I like this completeness, not only in the docs but also, that it's possible to be overwritten.

Reply:
Yes, the ability to override it comes for free thanks to `AbstractPowerManifold` :slightly_smiling_face: .